src/components/FlowRunCreateForm.vue

```diff
@@ -23,13 +23,7 @@
       </p-label>
 
       <p-label label="Tags (optional)">
-        <p-tags-input v-model="tags" :options="deploymentTags">
-          <template #default="{ selectedOption }">
-            <p-tag :dismissible="!selectedOption.disabled">
-              {{ selectedOption.label }}
-            </p-tag>
-          </template>
-        </p-tags-input>
+        <p-tags-input v-model="tags" :options="deploymentTags" />
```

Comment:
Ah is this what the update to `p-tags-input` was for?

Reply:
Yeah exactly haha